src/ui/components/DismissibleTextForm/index.js

```diff
@@ -211,14 +264,14 @@ export class DismissibleTextFormBase extends React.Component<
     return (
       <form className={makeClassName('DismissibleTextForm-form', className)}>
         <Textarea
-          defaultValue={this.state.text}
```

Comment:
I honestly don't know how this worked before but it did. I read the source twice. I have no idea! For some reason, it no longer works with my new call to `setState({ text: ... })` but I don't know what's different. In any case, it should have always been setting the `value` prop, not `defaultValue`.